src/node_crypto.cc

```diff
@@ -2319,18 +2319,13 @@ int SSLWrap<Base>::SSLCertCallback(SSL* s, void* arg) {
 
   Local<Object> info = Object::New(env->isolate());
 
-  SSL_SESSION* sess = SSL_get_session(s);
-  if (sess != nullptr) {
-    if (sess->tlsext_hostname == nullptr) {
-      info->Set(env->servername_string(), String::Empty(env->isolate()));
-    } else {
-      Local<String> servername = OneByteString(env->isolate(),
-                                               sess->tlsext_hostname,
-                                               strlen(sess->tlsext_hostname));
-      info->Set(env->servername_string(), servername);
-    }
-    info->Set(env->tls_ticket_string(),
```

Comment:
`tls_ticket_string` is still used in `SSLWrap<Base>::OnClientHello`. I believe at that point it is meaningful.

The reference to `tlsTicket` in `_tls_wrap.js` is to the one set in `OnClientHello` and should continue to work.